conf/onwire.h

```diff
@@ -224,10 +224,13 @@ struct m0_confx_fdmi_flt_grp {
 
 struct m0_confx_fdmi_filter {
 	struct m0_confx_header xf_header;
+	uint32_t               xf_type;
```

Comment:
Can we:
```
uint32_t xf_type M0_XCA_FENUM(enum m0_fdmi_filter_type_id);
```
?

Reply:
`enum` word is not needed inside `M0_XCA_FENUM()`. The change also requires
```
diff --git a/fdmi/filter.h b/fdmi/filter.h
index 81b7f1bd..0da56f31 100644
--- a/fdmi/filter.h
+++ b/fdmi/filter.h
@@ -78,7 +78,7 @@ enum m0_fdmi_filter_type_id {
        M0_FDMI_FILTER_TYPE_TREE = 1,
        /** Substring-based filter for CAS kv pairs */
        M0_FDMI_FILTER_TYPE_KV_SUBSTRING,
-};
+} M0_XCA_ENUM;
 
 /**
  * Filter operation code
```

Updated in #719.